cli.js

```diff
@@ -17,4 +17,14 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, function (err) {
```

Comment:
I assumed that. But in that case, without `try`...`catch`, missing `src` and `dest` errors will output that way:
```sh
$ ./cli.js 

…/cpy/index.js:9
		throw new Error('`src` and `dest` required');
		      ^
Error: `src` and `dest` required
    at module.exports (…/cpy/index.js:9:9)
    at Object.<anonymous> (…/cpy/cli.js:20:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:929:3
```